### PR TITLE
Audrey Teo Homework 1

### PR DESCRIPTION
**Pull request by audreyteo**
The most challenging part of the whole process was figuring the push/pull/commit. The trouble I kept running into at first was pushing before committing so my changes would be not showing up on the online repository.

The R Markdown syntax is fairly straightforward especially because there is so much documentation online.

### CODE REVIEW

**Comment by Ed-San1** [comment]
Evaluation
----------------------------------------------------
| HW1                         | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |   X        |              |            |
| **Ease of access**          |     X      |              |            |

- Good job leaving a clear description of every change you made throughout the process.
- Very interesting choices for examples of data visualization, I looked into the W.E.B. Du Bois one and found that there's a whole set of them. 
- Just as a comment, you only did this with the picture of your dog but you don't necessarily have to save an image with the project file. You can upload the image to a hosting site (such as imgur) and then just link it.

**Review by hehely — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

- Successfully used all the required elements in writing the biography

- Code is neat and easy to understand

**Comment by rrodriguezbarron** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |   x        |              |            |

Remarks:

* This is a very good homework. I commend you for going above the requirements using comments that are helpful to understand your code.

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About me
+
+My name is *_Audrey Teo_*. I'm a fourth-year student majoring in _Public Policy Studies_. I'm taking this class because I'm currently working on my B.A. Thesis and think this class would be very helpful when I start working with data. 
+
+### Few examples of data visualization that I've encountered in the past that I think are cool
+1. W.E.E.B. Du Bois's [Infographics of African American Life in the 1900s](https://publicdomainreview.org/collections/w-e-b-du-bois-hand-drawn-infographics-of-african-american-life-1900/)
```
Comment (rrodriguezbarron) [inline README.md:8]: This is very good

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |   x        |              |            |

Remarks:

* This is a very good homework. I commend you for going above the requirements using comments that are helpful to understand your code.

**Pull request closed without merging**